adafruit_circuitplayground/express.py

```diff
@@ -658,10 +658,41 @@ def stop_tone(self):
         # Stop playing any tones.
         if self._sample is not None and self._sample.playing:
             self._sample.stop()
-            self._sample.deinit()
-            self._sample = None
```

Comment:
Why are these two lines deleted?

Reply:
undesired pause added at the end of tones

should i add back a command to deinit() the pin? it is useful but shouldnt be called willy nilly